package.json

```diff
@@ -6,8 +6,9 @@
     "prettier": "@pxblue/prettier-config",
     "scripts": {
         "initialize": "bash scripts/initializeSubmodule.sh",
-        "test": "cd ./components && yarn test:ci",
+        "test": "cd ./components && yarn test",
```

Comment:
Is there a reason from switching from `yarn test:ci` (without watcher) to `yarn test` here? I know we originally disabled the watcher because the tests would hang forever in circleCI, but it looks like they passed for this branch, so just wondering.

Reply:
I'd use `test:ci` in the circleci, and update the command to be `yarn test --no-watch --code-coverage`.  That way the `yarn coverage` command can be removed. 

Other than that, branch looks good to me!

Reply:
No reason was from testing local and to keep karma window open after tests complete. so its back in

Reply:
> I'd use `test:ci` in the circleci, and update the command to be `yarn test --no-watch --code-coverage`. That way the `yarn coverage` command can be removed.
> 
> Other than that, branch looks good to me!

not sure what you mean here. Use Test:ci as the call command to execute or do yarn test:ci --code-coverage...
yarn test:ci --code-coverage doesn't work.
We can make it anything but no-watch in CI is the same as test:ci locally which just opens/closes karma

Reply:
the `yarn test:ci` command should be called only within circle ci, I'm not sure why it's inheriting the `--coverage` and `--no watch` flag instead of just hard-coding those flags directly into the script.  Instead of being able to look at the package.json to see what circle ci is running for the test stage, you have to look at both the .circle config + package.json  to get the full picture of what's happening.

Reply:
Also for some reason, I thought the coverage script was a duplicated script, but that was my mistake.  The `yarn test:ci` and `yarn coverage` script can coexist.

Reply:
ok we should be good now. We are calling test:ci for CCI and coverage too